### PR TITLE
Kuc unit test label

### PR DESCRIPTION
**Pull request by minhhy2801**

### CODE REVIEW

File: src/js/Label/__test__/setTextColor.test.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+  test('setTextColor successfully with full props Label component', () => {
+    const label = new Label({text: 'This is Label', backgroundColor: 'red', isDisabled: true, isRequired: true, isVisible: true, textColor: 'blue'});
+    label.setTextColor('red');
+    expect((label.render().firstElementChild as HTMLElement).style.color).toBe('red');
+  });
+
+  test('setTextColor successfully with props value number Label component', () => {
```
Comment (phongnhat19) [inline src/js/Label/__test__/setTextColor.test.ts]: Description is incorrect, expect to throw error.

File: src/react/Label/__test__/onClick.test.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    console.error.mockImplementation(() => { });
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('onClick without props Label', () => {
+    const onClick = (e: React.SyntheticEvent<EventTarget>) => {
+      expect((e.target as HTMLInputElement).getElementsByTagName('span')[0]).toBeTruthy();
```
Comment (phongnhat19) [inline src/react/Label/__test__/onClick.test.tsx]: Only need to `expect(true)`

File: src/react/Label/__test__/render.test.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+  });
+  test('render with full props Label', () => {
+    const {container} = render(<Label backgroundColor="red" isRequired isVisible text="label" textColor="blue" />);
+    if (container.firstElementChild) {
+      expect(container.firstElementChild.className).toBe('kuc-label');
+    }
+  });
+
+  test('render with props isVisible props Label', () => {
```
Comment (phongnhat19) [inline src/react/Label/__test__/render.test.tsx]: Unnecessary test case

File: src/js/Label/__test__/render.test.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('render successfully without props Label component and enable', () => {
+    const label = new Label();
+    expect(label.render().className).toBe('kuc-label');
+  });
+  test('render successfully with props value Label component', () => {
```
[the remark this answers is not in the document]
Comment (minhhy2801) [reply src/js/Label/__test__/render.test.ts]: Screen Shot 2019-12-25 at 14 30 54

File: src/react/Label/__test__/render.test.tsx (hunk 3 above)
Comment (minhhy2801) [reply src/react/Label/__test__/render.test.tsx]: Screen Shot 2019-12-25 at 14 57 02

**Pull request merged**